packages/carbon-web-components/src/components/slug/slug.scss

```diff
@@ -135,200 +182,217 @@
+:host(
+    #{$prefix}-slug:not([kind='hollow']):not(
+        [dot-type='hollow']
+      )[alignment^='left']
+  ) {
+  .#{$prefix}--popover-caret {
+    &::before {
+      inset-inline-end: to-rem(1px);
+      transform: rotate(135deg);
+    }
 
-    .#{$prefix}--slug__text::before {
-      background-color: $tag-color-gray;
+    &::after {
+      border-end-end-radius: 50%;
+      border-start-end-radius: 50%;
+      inset-block-start: to-rem(-1px);
+      inset-inline-start: $spacing-01;
     }
   }
 }
 
-:host(#{$prefix}-slug[tag='cool-gray']) {
-  .#{$prefix}--slug__text {
-    color: $tag-color-cool-gray;
-  }
-
-  .#{$prefix}--slug__text::before {
-    background: $tag-color-cool-gray;
+// override for specific alignments
+:host(
+    #{$prefix}-slug:not([kind='hollow']):not(
+        [dot-type='hollow']
+      )[alignment='left-bottom']
+  ),
+:host(
+    #{$prefix}-slug:not([kind='hollow']):not(
+        [dot-type='hollow']
+      )[alignment='right-bottom']
+  ) {
+  .#{$prefix}--popover-caret::after {
+    background: transparent;
   }
+}
 
-  button:hover {
-    border-color: $tag-color-cool-gray;
-
-    .#{$prefix}--slug__text::before {
-      background-color: $tag-color-cool-gray;
-    }
+// Change caret color / border when it is near the bottom of the callout
+:host(
+    #{$prefix}-slug:not([kind='hollow']):not(
+        [dot-type='hollow']
+      )[alignment='left-bottom']
+  ),
+:host(
+    #{$prefix}-slug:not([kind='hollow']):not(
+        [dot-type='hollow']
+      )[alignment='right-bottom']
+  ),
+:host(
+    #{$prefix}-slug:not([kind='hollow']):not(
+        [dot-type='hollow']
+      )[alignment='top']
+  ),
+:host(
+    #{$prefix}-slug:not([kind='hollow']):not(
+        [dot-type='hollow']
+      )[alignment='top-right']
+  ),
+:host(
+    #{$prefix}-slug:not([kind='hollow']):not(
+        [dot-type='hollow']
+      )[alignment='top-left']
+  ) {
+  .#{$prefix}--popover-caret::before {
+    border-color: $slug-callout-caret-bottom;
+    background: $slug-callout-caret-bottom-background;
   }
 }
 
-:host(#{$prefix}-slug[tag='warm-gray']) {
-  .#{$prefix}--slug__text {
-    color: $tag-color-warm-gray;
+// Adjust background if actions are present
+:host(
+    #{$prefix}-slug:not([kind='hollow']):not(
+        [dot-type='hollow']
+      )[alignment='left-bottom'][has-actions]
+  ),
+:host(
+    #{$prefix}-slug:not([kind='hollow']):not(
+        [dot-type='hollow']
+      )[alignment='right-bottom'][has-actions]
+  ),
+:host(
+    #{$prefix}-slug:not([kind='hollow']):not(
+        [dot-type='hollow']
+      )[alignment='top'][has-actions]
+  ),
+:host(
+    #{$prefix}-slug:not([kind='hollow']):not(
+        [dot-type='hollow']
+      )[alignment='top-right'][has-actions]
+  ),
+:host(
+    #{$prefix}-slug:not([kind='hollow']):not(
+        [dot-type='hollow']
+      )[alignment='top-left'][has-actions]
+  ) {
+  .#{$prefix}--popover-caret::before {
+    background: $slug-callout-caret-bottom-background-actions;
   }
+}
```

Comment:
```suggestion
```

I added this to the suggestion directly above. GH wouldn't let me edit both blocks at the same time.